build.gradle

```diff
@@ -201,6 +201,7 @@ intellijPlatformTesting.runIde {
                         "-Dapple.laf.useScreenMenuBar=false",
                 ]
             } as CommandLineArgumentProvider)
+            systemProperty("ide.native.launcher", "true")
```

Comment:
In order to get the widows build success , I just picked the change of the PR https://github.com/OpenLiberty/liberty-tools-intellij/pull/1147/files , once this PR is merged, I can remove this change.

Reply:
@vaisakhkannan You can remove this code snippet from here. Once the PR https://github.com/OpenLiberty/liberty-tools-intellij/pull/1147/files merge, you can pull the latest and update the PR

Reply:
<img width="1231" alt="Screenshot 2024-12-04 at 6 00 27 PM" src="https://github.com/user-attachments/assets/4bd4e711-8168-4e4c-b658-3e8d46f7b203">

Thanks @dessina-devasia , Just added a screenshot that both windows and linux are passing the build with this.

I am removing `systemProperty("ide.native.launcher", "true")` as part of suggestion.

Reply:
@dessina-devasia , I rebased the branch with the latest changes because I merged https://github.com/OpenLiberty/liberty-tools-intellij/pull/1147.